src/services/user/services/QrRegistrationLinks.js

```diff
@@ -13,40 +13,97 @@ class QrRegistrationLinks {
 	}
 
 	async create(data) {
-		let { userIds } = data;
 		try {
-			userIds = await this.getUsersIdsWithoutAccount(userIds);
-			if (!userIds.length) {
+			const validUserIds = await this.getValidUserIds(data);
+			const userWithoutAccount = await this.getUsersIdsWithoutAccount(validUserIds);
+			if (!userWithoutAccount.length) {
 				return [];
 			}
-			return this.generateQrRegistrationLinks(userIds);
+			const generateServiceLinkParams = this.getGenerateLinkServiceParams(data);
```

Comment:
This naming 'generateServiceLinkParams' is used in serveral places. At this places this name suggest to me, that it is some method, wherease the data is already there. More proper naming for me would be
'generate**d**ServiceLinkParams'. What do you think ?